azure-pipelines-official.yml

```diff
@@ -1,42 +1,99 @@
-trigger:
-- master
+pool:
+  name: VSEng-MicroBuildVS2017
+  demands: msbuild
 
 variables:
   solution: 'MSBuildPrediction.sln'
   buildPlatform: 'Any CPU'
   buildConfiguration: 'Release'
+  signType: 'Test'
+  teamName: 'MSBuild'
 
-jobs:
-- job: BuildOnWindows
-  displayName: "Build and test on Windows"
-  pool:
-    name: 'Hosted Windows 2019 with VS2019'
-  steps:
-  - task: MSBuild@1
-    inputs:
-      solution: '**/*.sln'
-      msbuildVersion: '16.0'
-  - task: VSTest@2
-    inputs:
-      testSelector: 'testAssemblies'
-      testAssemblyVer2: | 
-        src\**\*test*.dll
-        !src\**\obj\**
-      searchFolder: '$(System.DefaultWorkingDirectory)'
-      runOnlyImpactedTests: false
-      runInParallel: true
-      codeCoverageEnabled: true
-- job: BuildOnLinux
-  displayName: "Build and test on Linux"
-  pool:
-    name: 'Hosted Ubuntu 1604'
-  steps:
-  - task: DotNetCoreCLI@2
-    inputs:
-      command: 'build'
-      projects: 'MSBuildPrediction.sln'
-# Test not passing on Linux
-#  - task: DotNetCoreCLI@2
-#    inputs:
-#      command: 'test'
-#      projects: 'MSBuildPrediction.sln'
+steps:
+- task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+  displayName: 'Install Signing Plugin'
+  inputs:
+    signType: '$(SignType)'
+
+- task: MSBuild@1
+  inputs:
+    solution: '$(solution)'
+    msbuildVersion: '16.0'
+    msbuildArguments: '/bl'
+
+- task: CopyFiles@2
+  displayName: Collect Symbols
+  inputs:
+    SourceFolder: 'src/'
+    Contents: '**\*.pdb'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/symbols'
+    CleanTargetFolder: true
+
+- task: PublishBuildArtifacts@1
+  displayName: Upload Symbols
+  condition: eq(variables['system.pullrequest.isfork'], false)
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)/symbols'
+    ArtifactName: symbols
+    ArtifactType: Container
+
+- task: CopyFiles@2
+  displayName: Collect Binaries
+  inputs:
+    SourceFolder: 'src/BuildPrediction/bin'
+    Contents: '**'
```

Comment:
(Not sure) should this just be dlls?

Reply:
Nvm, this is the "main output" i guess.

---
In reply to: [292087563](https://github.com/microsoft/MSBuildPrediction/pull/14#discussion_r292087563) [](ancestors = 292087563)